src/api/readCpmProduct/src/v1/steps.py

```diff
@@ -0,0 +1,68 @@
+from aws_lambda_powertools.utilities.data_classes import APIGatewayProxyEvent
+from domain.repository.cpm_product_repository.v3 import CpmProductRepository
+from domain.response.validation_errors import mark_validation_errors_as_inbound
+from event.step_chain import StepChain
+from pydantic import BaseModel, Field
+
+
+class PathParametersModel(BaseModel):
```

Comment:
Could be worth moving to a layer as other lambda may need to use it. e.g.
CreateDeviceReferenceData POST /ProductTeam/{product_team_id}/Product/{product_id}/DeviceReferenceData

Reply:
maybe worth a huddle, I think it would be worth having a folder in a layer for api params (separate from the domain/core models) and having different models for different objects: